tests/spec/invalid.coffee

```diff
@@ -89,3 +89,20 @@ describe "Invalid case tests", ->
       ]).then(cb).catch(
         (err) -> throw err
       ).done (-> done()), done
+
+describe "Print Java error message", ->
+  it "The message should be printed when `validate()` called", ->
```

Comment:
You can use `describe` function in `describe` function. Therefore, you can write the following:

```coffeescript
describe "When `validate()` is called, ", ->
  it "The message should be printed", ->
    ....
```